TYPO3.Neos/Resources/Private/Templates/Media/Asset/RelatedDocumentNodes.html

```diff
@@ -0,0 +1,28 @@
+{namespace m=TYPO3\Media\ViewHelpers}
+{namespace neos=TYPO3\Neos\ViewHelpers}
+<f:layout name="EditImage"/>
+
+<f:section name="Title">RelatedDocumentNodes view of AssetController</f:section>
+
+<f:section name="Content">
+  <div class="neos-row-fluid">
+    <div class="neos-span6">
+      <legend>{asset.resource.filename}</legend>
+
+      <table class="neos-info-table">
+        <tbody>
+          <f:for each="{relatedDocumentNodes}" as="node">
+            <tr>
+              <td><neos:link.node node="{node}" /></td>
+              <td class="neos-aRight">{node.nodeData.lastModificationDateTime -> f:format.date(format: 'Y-m-d H:i')}</td>
+            </tr>
+          </f:for>
```

Comment:
it might be useful to see the actual node and not just the document, maybe it could be nested underneath.. we can't link to the node atm., but that should be possible in the future.. what do you think?